dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -567,6 +569,19 @@ private Response getFileByToolVersionID(String registryId, String versionIdParam
         return Response.status(status).build();
     }
 
+    public static List<Checksum> getFileDescriptorChecksumsForTrs(final SourceFile sourceFile) {
```

Comment:
I think you should rename this method to `getSourceFileChecksums` or `getSourceFileChecksumsForTrs` (I prefer the former). I think this caused the confusion on my other comment -- you're calling a method that says get file descriptor checksums, but then pass it a Dockerfile.

Reply:
On further reflection, `...forTrs` is OK; I skimmed it the first time, not realizing the checksum is represented in two ways.